### PR TITLE
Bump to jenkins 2.235.3

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

**Comment by ramereth** [comment]
@TheREK3R can you please rebase and resolve the conflicts on this?

**Comment by TheREK3R** [comment]
oops, should be fixed now

File: test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@
   it { should be_executable }
 end
 
-describe gem('openstack_taster', "/opt/#{chef}/embedded/bin/gem") do
+describe gem('openstack_taster', '/opt/${chef}/embedded/bin/gem') do
```
Comment (ramereth) [inline test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb:54]: This is incorrect. Why was this changed?

Comment (TheREK3R) [reply test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb:54]: oh shoot, it got flagged by cookstyle for some reason

Comment (TheREK3R) [reply test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb:54]: oh , what happened I think is that the $ was in my version, but then on rebase it changed and i applied the single quotes in the merge

**Comment by TheREK3R** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.8.3.  Have a nice day!
